.github/workflows/check-published-scheduled.yml

```diff
@@ -10,8 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     name: Check Published
     steps:
-      - name: Checkout Code
-        uses: actions/checkout@v4
+      - uses: actions/checkout@v4
```

Comment:
add back 'name'?

Reply:
i just re-copied everything from package standards, should be good now.